qa/rpc-tests/wallet_shieldcoinbase_sapling.py

```diff
@@ -8,7 +7,7 @@
 WalletShieldCoinbaseTest = object
 
 cwd = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
-execfile(os.path.join(cwd, 'wallet_shieldcoinbase.py'))
+exec(open(os.path.join(cwd, 'wallet_shieldcoinbase.py')).read())
```

Comment:
Why not use a conventional import statement?


Everything from line 3 to line 10 can _and should_ be replaced with a much simpler and more readable:

`from wallet_shieldcoinbase import WalletShieldCoinbaseTest`

unless there's compelling reason for the less readable version.